frontend/src/metabase/app.js

```diff
@@ -70,12 +71,16 @@ function _init(reducers, getRoutes, callback) {
   root.render(
     <Provider store={store}>
       <EmotionCacheProvider>
-        <DragDropContextProvider backend={HTML5Backend} context={{ window }}>
-          <ThemeProvider>
-            <GlobalStyles />
-            <Router history={history}>{routes}</Router>
-          </ThemeProvider>
-        </DragDropContextProvider>
+        <DragDropContextProviderContext.Provider
```

Comment:
I'm using this to check if we should be using the draggable collection item or just a regular list item. We can't use the draggable items in the SDK since we don't use the react dnd library - if we don't have a `DragDropContextProvider`, any component that relies on it just doesn't render